### DISCUSSION TITLE
Switching accounts

### DISCUSSION THREAD

**Discussion by jpvoyd** [abraunegg/onedrive#1537] [Q&A] [answered]
Hi all... I'm going nuts here: I tried logging out of my current account and log back in with the credentials of my new employer. Now it says that I'm logged off, but it's still syncing to the old account. Then I figured I might us well update to the latest version. This doesn't work either. So I figured what the heck, I just remove it and reinstall it then configure it with the new credentials. Now it says the package has been removed: but it's STILL syncing to the old account. It's mysterious as hell. So this is "the" problem. I guess it can be summed up for now: how in hell can I remove this package properly (to later reinstall)?

**Comment by abraunegg** [comment, marked as answer]
@jpvoyd 
You are missing some critical information here ... such as:
* What Linux OS are you running?
* What application version did you install and from where?

Assumptions based on the lack of providing these details, is that you are a Ubuntu user, and you have installed the package from the PPA or default repositories, which is impacted by this bug: https://github.com/abraunegg/onedrive/issues/1274

How to solve:
* Dont use Ubuntu or any Ubuntu based Linux OS
* Compile and install the application from source

When configuring the application, configure it so you can use multi-accounts correctly by following: https://github.com/abraunegg/onedrive/blob/master/docs/advanced-usage.md

**Reply by jpvoyd** [reply]
Thank you very much for your swift reply. Yes, you guessed correctly about my distro. I have now updated onedrive to 2.4.12 (compiled and installed from source). I will look at bug #1274 and consider your other suggestions. Thanks again.
